torch_geometric/nn/conv/cugraph/gat_conv.py

```diff
@@ -75,10 +78,12 @@ def forward(
 
         if LEGACY_MODE:
             out = GATConvAgg(x, self.att, graph, self.heads, 'LeakyReLU',
-                             self.negative_slope, False, self.concat)
+                             self.negative_slope, False, self.concat,
+                             edge_feat=edge_attr)
         else:
             out = GATConvAgg(x, self.att, graph, self.heads, 'LeakyReLU',
-                             self.negative_slope, self.concat)
+                             self.negative_slope, self.concat,
+                             edge_feat=edge_attr)
```

Comment:
Looks like a user-facing change. Mind updating changelog and adding a quick test case?